LayoutTests/TestExpectations

```diff
@@ -6938,7 +6938,8 @@ imported/w3c/web-platform-tests/navigation-api/scroll-behavior/scroll-on-synthet
 imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/ [ Pass ]
 imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-away-and-back.html [ Skip ]
 imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-location-api.html [ Skip ]
-imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/location-reload.html [ Pass ]
+imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-meta-refresh.html [ Pass ]
+imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-reload.html [ Pass ]
```

Comment:
I really think we should merge https://github.com/WebKit/WebKit/pull/27818 first.